### ISSUE TITLE
NVIDIA CUDA GPU programming — processing 8 BILLION hashes/sec on V100s

### ISSUE THREAD

**Issue by Rimatara** [intel/llvm#6943]
How to modify the program to run it on Windows 10 with Visual Studio 2017 and installed NVIDIA CUDA support and OneAPI?

See the program running on Linux by using NVCC compiler on [Youtube link here](https://www.youtube.com/watch?v=6AXl0sGjxg0) (Github link to source files [in the description below](https://github.com/tanmayb123/redesigned-train)).

I would like to try to run it on my ASUS UX510UWK notebook (i7-7500U, 24GB RAM, Samsung SSD 980 PRO 1TB) with NVIDIA GeForce GTX 960M.

**Comment by zjin-lcf** [comment]
Do you imply that the SYCL program couldn't reach 8 billion hashes per second on the GPU ?

**Comment by Rimatara** [comment]
No. I am running the example above in Linux with NVIDIA CUDA support on my notebook already although I am not sure how to modify these parameters to suit my GTX 960M graphic card.

#define THREADS 1500
#define BLOCKS 256
#define GPUS 4

**Comment by zjin-lcf** [comment]
The question may be posted at https://community.intel.com/ for assistance
